sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -36,9 +36,18 @@ export class ApplicationApi extends HttpBaseClient {
 
   async getApplicationData(
     applicationId: number,
-    options?: { loadDynamicData?: boolean; isParentApplication?: boolean },
+    options?: {
+      studentId?: number;
+      loadDynamicData?: boolean;
+      isParentApplication?: boolean;
+    },
   ): Promise<ApplicationDataAPIOutDTO | ApplicationSupplementalDataAPIOutDTO> {
-    let url = this.addClientRoot(`application/${applicationId}`);
+    let url = this.addClientRoot("application");
+    if (options?.studentId) {
+      url = `${url}/student/${options?.studentId}/application/${applicationId}`;
```

Comment:
`${options?.studentId}` does not require null propagation and can have the "?" removed as `${options.studentId}`. Same for the below one that is not part of this PR.
```ts
url = `${url}?loadDynamicData=${options.loadDynamicData}`
```